### DISCUSSION TITLE
🌟 Should I Start Uploading LeetCode POTD?

### DISCUSSION THREAD

**Discussion by Hunterdii** [Hunterdii/GeeksforGeeks-POTD#15] [Polls]
Hey, amazing community! 😄

I've been posting GeeksforGeeks POTD solutions consistently, but now I’m thinking about starting LeetCode POTD as well! What do you think?

**Comment by Hunterdii** [comment]
✨ Expand to other platforms like:  
- **CodeChef Daily Challenges**  
- **HackerRank Problem of the Day**  
- **Codeforces Daily Problem**  
- **AtCoder Practice Problems**  

💡 **Not interested in daily challenges? Suggest other content (e.g., topic-wise solutions, Any-other Contests, etc.) in the comments!**  

Head over to the [discussion poll](https://github.com/Hunterdii/GeeksforGeeks-POTD/discussions/15) and make your voice heard! 🌟

**Reply by VoidSon-Stubs** [reply]
Start Codechef if possible 😄
